packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -64,6 +96,15 @@ export default function TemplatePanel() {
 			>
 				<TemplateAreas />
 			</SidebarCard>
+			<p>
+				{ __(
+					'Choose a predefined pattern to switch up the look of your template.' // TODO - make this dynamic?
```

Comment:
I think this can be shortened.
But I also think the switch context is important so that the user doesn't think it will just place another pattern in the editor.